webextensions/api/windows.json

```diff
@@ -766,7 +766,7 @@
               "__compat": {
                 "support": {
                   "chrome": {
-                    "version_added": true
+                    "version_added": "≤61"
```

Comment:
Wasn't this (and top, type, and width) added in [302](https://github.com/mdn/browser-compat-data/pull/307/files) - e.g. ≤60